semantic/src/site/globals/site.overrides

```diff
@@ -2688,14 +2692,20 @@ h6:first-child {
 
   #menu-bar {
     font-size: inherit;
+    position: absolute;
+    top: @mobileHeaderHeight;
+    left: 0px;
+    width: 100%;
+    background-color: inherit;
```

Comment:
The `inherit` makes the list take the same color as the main header bar (doesn't work properly here, but does in #1591